vtex/styleguide
# fix: InputSearch icon unaligned

#### What is the purpose of this pull request?

Fixes ##1302.

#### What problem is this solving?

keeps the search Icon vertically aligned, as it should be

#### How should this be tested manually?

open the InputSearch component and see if the icons are centered horizontally and vertically

#### Screenshots or usage example
before:
![image](https://user-images.githubusercontent.com/44899195/91058123-7c6a0c00-e5fe-11ea-8c6f-95fff170fe44.png)


after:
![image](https://user-images.githubusercontent.com/44899195/91056977-01542600-e5fd-11ea-8781-d2d462b67fb9.png)

#### Types of changes

- [x] Bug fix (an uninterrupted change that fixes a problem)
- [] New feature (a nonstop change that adds functionality)
- [] Breaking the change (fix or feature that would cause the existing functionality to be changed)
- [] Requires changes to the documentation, which has been updated accordingly.
